packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -55,34 +62,24 @@ $skeleton-color-end: rgba($gray1, 0.2) !default;
   }
 }
 
-// Some properties should not be overriden. A more specific explanation is given before each usage
-// of "!important".
+// This class hides content with a glowing, rounded rectangle. CSS properties that should not be overriden by consumers use the
```

Comment:
i still think the wording is backwards. it's not that consumers shouldn't override these styles but rather that these styles _must_ override all consumer styles.